delta/plugins/composite-views/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/compositeviews/model/CompositeViewRejection.scala

```diff
@@ -13,7 +13,9 @@ import io.circe.Json
   *
   * @param reason a descriptive message as to why the rejection occurred
   */
-sealed abstract class CompositeViewRejection(val reason: String) extends Product with Serializable
+sealed trait CompositeViewRejection extends Product with Serializable {
```

Comment:
Is there a reason why you did that in a different way than what we have for all the other rejections?

Look at `ElasticSearchViewsRejection` for example:
```scala
sealed abstract class ElasticSearchViewRejection(val reason: String) extends Product with Serializable

```

Reply:
I started like this but had to change it, because abstract class doesn't work with 
```scala
 sealed trait CompositeViewSourceRejection extends CompositeViewRejection
```

Reply:
Well, just make it an abstract class. We have an example of `ProjectRejection` that is done in this way:

```scala
sealed abstract class ProjectRejection(val reason: String) extends Product with Serializable

object ProjectRejection {

  sealed abstract class NotFound(reason: String) extends ProjectRejection(reason)
...
```